### PR TITLE
adding comment about rsa

### PR DESCRIPTION
**Pull request by choldgraf**

### CODE REVIEW

File: examples/decoding/decoding_rsa.py [hunk 1]
```diff
@@ -12,7 +12,8 @@
 In this example, we perform RSA on responses to 24 object images (among
 a list of 92 images). Subjects were presented with images of human, animal
 and inanimate objects [4]_. Here we use the 24 unique images of faces
-and body parts.
+and body parts. Note: this example will download a very large (~6GB) file,
+so we will not build the images below.
```
Comment (Eric89GXL) [inline examples/decoding/decoding_rsa.py]: this looks better as:
```
.. note:: whatever
```

Comment (agramfort) [inline examples/decoding/decoding_rsa.py]: +1

**Comment by choldgraf** [comment]
how's that?

**Comment by agramfort** [comment]
thx @choldgraf

**Comment by choldgraf** [comment]
:gong:

**Pull request merged**